### ISSUE TITLE
Doesn't work with "Shiver"

### ISSUE THREAD

**Issue by ExtemTheHedgepog** [cabalex/fnf-song-converter#2]
I tried to port "Shiver", a "third-song-for-spooky-month mod", using the Converter. It works for the most part, but the Spooky Kids' notes don't exist and BF's notes cut off close to the end of the song. How do I fix it?

**Comment by cabalex** [comment]
Fixed this with the newest commit! The issue was with how I compiled section and note timings- the converter always expected the notes to be within the sections' timings. However, with songs like this, there are two sections used (one for each player), and both contain notes way before the section is normally played (e.g. the section is at TIME 80,000, but the first note in the section is at TIME 6,000), meaning that the game wouldn't create the notes until way after it was supposed to.

This fix should also mean that other songs that use this kind of beatmap should work too (e.g. some of the B side songs). Thanks for reporting it!

![image](https://user-images.githubusercontent.com/31020729/118747713-5aa95880-b80f-11eb-9e87-6a545d2a760e.png)
